region/wreckedship/main.json

```diff
@@ -2180,17 +2187,37 @@
                   "name": "Bomb the blocks",
                   "notable": false,
                   "requires": ["h_canBombThings"],
-                  "note": "If using PBs, doable with just one by placing it far enough right."
+                  "note": "If using PBs, place it far enough right to reach the shot blocks."
                 },
                 {
                   "name": "Wave Screw",
                   "notable": false,
                   "requires": [
                     "Wave",
+                    "ScrewAttack",
+                    "Morph",
+                    {"or": [
+                      "Charge",
+                      "Plasma",
+                      "Spazer"
+                    ]}
+                  ],
+                  "note": "The shot blocks can be broken with Wave and any of charge, spazer or plasma by standing at the top of the stairs and firing diagonally down.",
+                  "devNote": "A very precise wave shot can work but is much harder than the beamless strat."
```

Comment:
These two strats seem to be split based on knowledge of the strat vs just attempting it and having it work. The problem is that wave+charge is somewhat unintuitive to me, so it seems like you would need to know about the strat to use that as well. Similarly wave+any of the three all seem to be fairly finicky. Most places in the room dont actually work when the camera is locked, so a player would either need to get lucky or know about the strat. 
TLDR: I think you need to know about this strat in order to perform it, and because of that, I think only the notable camera unlock strat should be in logic.